zetaclient/orchestrator/orchestrator.go

```diff
@@ -518,3 +565,60 @@ func (oc *Orchestrator) ScheduleCctxBTC(
 		}
 	}
 }
+
+// runObserverSignerSync runs a blocking ticker that observes chain changes from zetacore
+// and optionally (de)provisions respective observers and signers.
+func (oc *Orchestrator) runObserverSignerSync(ctx context.Context) error {
+	const cadence = 5 * time.Second
```

Comment:
If you want to use a constant, `ZetaBlockTime` (or `ZetaBlockTime * 2`) are good options as the chain state (chains, parameters are part of chain state) changes block by block.